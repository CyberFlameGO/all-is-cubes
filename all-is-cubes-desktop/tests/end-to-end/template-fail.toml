bin.name = "all-is-cubes"
args = "--template fail --graphics headless"
status.code = 1
