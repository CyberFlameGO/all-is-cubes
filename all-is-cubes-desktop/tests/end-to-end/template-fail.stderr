...
Error: An error occurred while generating object 'space'

Caused by:
    the Fail template always fails to generate
